src/objects/conf/base.py

```diff
@@ -347,12 +352,12 @@
 # If True only locks based on user id and never locks by IP if attempts limit
 # exceed, otherwise utilize the existing IP and user locking logic Default:
 # False
-AXES_ONLY_USER_FAILURES = True
+AXES_LOCKOUT_PARAMETERS = ["username"]
 # If set, specifies a template to render when a user is locked out. Template
 # receives cooloff_time and failure_limit as context variables. Default: None
 AXES_LOCKOUT_TEMPLATE = "account_blocked.html"
-AXES_USE_USER_AGENT = True  # Default: False
-AXES_LOCK_OUT_BY_COMBINATION_USER_AND_IP = True  # Default: False
```

Comment:
Why have you decided to change it?

Reply:
From Django Axes 6.0.0 these settings were deprecated: 

[Django Axes Changelog](https://django-axes.readthedocs.io/en/latest/10_changelog.html#id6)